matrix-meetings-widget/vitest.config.ts

```diff
@@ -16,10 +16,17 @@
 
 /// <reference types="vitest" />
 
+import path from 'path';
 import { defineConfig } from 'vite';
 
 export default defineConfig({
   test: {
+    alias: {
+      // solve "You are loading @emotion/react when it is already loaded…" issue
```

Comment:
I feel like we could also use this in neoboard :D there we see the same thihng :)